SQL/0000-00-00-schema.sql

```diff
@@ -1,3 +1,5 @@
+SET NAMES utf8;
```

Comment:
This is important to make sure strings ("Français" here) are sent as UTF8 to the database (even if the tables are already encoded as UTF8).